appdatabase/migrations/sql/1685880973_add_profile_links_settings_table.up.sql

```diff
@@ -0,0 +1,30 @@
+-- Create new table where we will store profile social links,
+-- after discussion https://github.com/status-im/status-go/pull/3552#issuecomment-1573817044
+-- we decided to replace the entire db table content on every change (because of that position column is not needed)
+CREATE TABLE IF NOT EXISTS profile_social_links (
+  text VARCHAR NOT NULL CHECK (length(trim(text)) > 0),
+  url VARCHAR NOT NULL CHECK (length(trim(url)) > 0),
+  PRIMARY KEY (text, url)
+);
```

Comment:
Yes, was thinking about the same, had `position` in previous implementation, but when I was changing the logic completely to what we've discussed (sending everything on each change) and since it was not added in an implementation which was here even before I completely forgot about it. Very well spotted, thanks! :)

Reply:
Will add it now.